src/pages/projects.html

```diff
@@ -58,16 +58,16 @@
 						<div class="list__item">
 							<div class="tile type_project">
 								<div class="tile__icon">
-									<div class="icon size_96x96 name_karma-openui5"></div>
+									<div class="icon size_96x96 name_ui5-uiveri5"></div>
 								</div>
-								<div class="tile__title"><a href="https://github.com/SAP/karma-openui5" class="link" rel="noopener" target="_blank">karma-openui5</a></div>
-								<div class="tile__text">Setup testing environment for your OpenUI5 projects with karma. </div>
+								<div class="tile__title"><a href="https://github.com/SAP/ui5-uiveri5" class="link" rel="noopener" target="_blank">UIVeri5</a></div>
+								<div class="tile__text">End-to-end testing framework for UI5-based applications.</div>
 							</div>
 						</div>
 					</div>
 					<div class ="section__bar">
 						<div class="bar__item">
-							<a href=" https://github.com/SAP?utf8=%E2%9C%93&q=openui5" class="link mod_offsite size_1" rel="noopener" target="_blank">More Official Projects on GitHub</a>
+							<a href=" https://github.com/SAP?utf8=%E2%9C%93&q=openui5" class="button size_2" rel="noopener" target="_blank">Browse More Official Projects</a>
```

Comment:
Not sure, it is good idea to convert a link to a button here and below.
It leads to more inconsistency.

Reply:
Technically, with this change it is not a link but an action - same as "download sdk" or "browse nightly" build. The link visualization for finding all projects is a bit to weak on this page imo

Reply:
rolled back